20231019/examples/valid/conceptsWithMultilingualLabels.json

```diff
@@ -0,0 +1,71 @@
+{
+  "@context": [
+    "https://w3id.org/kim/amb/context.jsonld",
+    {
+      "@language": "de"
+    }
+  ],
+  "name": "Arbeitsblatt - Mon avenir - Französisch - tutory.de",
+  "id": "https://www.tutory.de/w/fbbadf1a",
+  "image": "https://www.tutory.de/worksheet/fbbadf1a-145a-463d-9a43-1ae9965c86b9.jpg?width=1000",
+  "type": ["LearningResource"],
+  "creator": [
+    {
+      "name": "HerunterS",
+      "type": "Person"
+    }
+  ],
+  "description": "Französisch-Arbeitsblatt",
+  "about": [
+    {
+      "id": "https://w3id.org/kim/hochschulfaechersystematik/n059",
+      "prefLabel": {
+        "de": "Französisch",
+        "en": "French",
+        "fr": "Français"
+      },
+      "type": "Concept",
+      "inScheme": {
+        "id": "https://w3id.org/kim/hochschulfaechersystematik/scheme"
+      }
+    }
+  ],
+  "license": {
+    "id": "http://www.gnu.org/licenses/agpl-3.0"
+  },
+  "dateCreated": "2019-07-03",
+  "inLanguage": ["fr"],
+  "publisher": [
+    {
+      "type": "Organization",
+      "name": "Tutory",
+      "id": "https://www.tutory.de"
+    }
+  ],
+  "audience": [
+    {
+      "id": "http://purl.org/dcx/lrmi-vocabs/educationalAudienceRole/student",
+      "prefLabel": {
+        "en": "student",
+        "de": "Lernende"
+      },
+      "type": "Concept",
+      "inScheme": {
+        "id": "http://purl.org/dcx/lrmi-vocabs/educationalAudienceRole/"
+      }
```

Comment:
`inScheme` entfernen, da es auch in der Spec nicht mehr erwähnt wird?

Reply:
Fixed with [39adb10](https://github.com/dini-ag-kim/amb/pull/256/commits/39adb101cd746293be87af597cd4f2262f3e622e)